### PR TITLE
Reduce triggers for okd and other aws tests

### PR DESCRIPTION
**Pull request by patrickdillon**
This should reduce the number of unrelated tests being run in general.

### CODE REVIEW

**Comment by patrickdillon** [comment]
/cc @LorbusChris @vrutkovs 

This should trigger okd tests only when we're making changes to the fcos images. PTAL

File: ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml [hunk 1]
```diff
@@ -152,7 +152,7 @@ tests:
   optional: true
 - as: e2e-aws-ovn
   optional: true
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent|alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  run_if_changed: (fcos|okd)
```
Comment (LorbusChris) [inline ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml:155]: @patrickdillon should this've been?
```suggestion
  run_if_changed: (fcos|scos|okd)
```

Comment (vrutkovs) [reply ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml:155]: Fixing this in https://github.com/openshift/release/pull/45141

**Pull request merged**